packages/manager/modules/netapp/src/routing.js

```diff
@@ -0,0 +1,33 @@
+import { ListLayoutHelper } from '@ovh-ux/manager-ng-layout-helpers';
+
+export default /* @ngInject */ ($stateProvider) => {
+  $stateProvider.state('netapp.index', {
+    url: `?${ListLayoutHelper.urlQueryParams}`,
+    component: 'managerListLayout',
+    params: ListLayoutHelper.stateParams,
+    redirectTo: (transition) =>
+      transition
+        .injector()
+        .getAsync('resources')
+        .then((services) =>
+          services.length === 0
+            ? {
+                state: 'netapp.onboarding',
+              }
+            : false,
+        ),
+    resolve: {
+      ...ListLayoutHelper.stateResolves,
+      apiPath: () => '/storage/netapp',
```

Comment:
```suggestion
      apiPath: () => '/storage',
```

Reply:
Yes it is true for the schema, but not for the real API endpoint. So I've let it as is, and had overridden the schema resolve.